demos/can-util/input-focused.html

```diff
@@ -23,23 +23,24 @@
 
 <div>In focus: <b>{{inFocus}}</b></div>
 </script>
-	<script src="../../node_modules/steal/steal.js" main="@empty">
-		var stache = require("can-stache");
-		var DefineMap = require("can-define/map/map");
-		require("can-stache-converters");
 
-		var template = stache.from("demo-html");
-		var ViewModel = DefineMap.extend({
-			inFocus: "boolean",
-			putInFocus: function(){
-				this.inFocus = true;
-			}
-		});
-		var map = new ViewModel({ inFocus: false });
-		window.map = map;
+<script src="../../node_modules/steal/steal.js" main="@empty">
+var stache = require("can-stache");
+var DefineMap = require("can-define/map/map");
+require("can-stache-bindings");
```

Comment:
Switching from requiring `can-stache-converters` to just `can-stache-bindings` was the only change here.

Reply:
FYI `can-stache-converters` requires `can-stache-bindings`, but this is fine since no converters are being used.

Reply:
Yeah, which is why it was working, but I couldn’t see a reason why converters needed to be required…